### DISCUSSION TITLE
Shouldn't `vlang`, `alang`, and `slang` be in lowercase?

### DISCUSSION THREAD

**Discussion by WhoMI7** [tomasklaen/uosc#457] [General]
![image](https://user-images.githubusercontent.com/93109835/220110523-767a1e48-f8b7-432e-8bde-0d420603c599.png)
![image](https://user-images.githubusercontent.com/93109835/220110608-3cc8ba4a-d6fc-49c0-a22d-5045c178fcb4.png)
![image](https://user-images.githubusercontent.com/93109835/220110690-2ca5f417-8912-41b5-b138-6f44de91e539.png)
Just like in mkvmerge and mpv by default:

![image](https://user-images.githubusercontent.com/93109835/220111244-8217097b-b1eb-41d4-8e2b-4165c8f2824d.png)
![image](https://user-images.githubusercontent.com/93109835/220111308-af2346ed-10a4-4ba4-a312-22aeb72bdfb5.png)
![image](https://user-images.githubusercontent.com/93109835/220111333-d3b6d213-4f39-4fd0-9aed-205c0c2bece7.png)
![image](https://user-images.githubusercontent.com/93109835/220111368-fd1e8426-4946-4ba0-93c2-86e3a56352ac.png)

**Comment by tomasklaen** [comment]
I see that we are forcing it to be uppercase with `track.lang:upper()`. I think @christoph-heinrich made it this way, but regardless, it's the most important piece of information in that hint that everyone is looking for, so I don't see an issue with making it more prominent by uppercasing it.

**Reply by christoph-heinrich** [reply]
From what I remember the person requesting that additional information in the track menus included a screenshot from some other player that also had the language capitalized, so I simply mirrored that. I personally don't really care if it's capitalized or not.

Edit: Found it https://github.com/tomasklaen/uosc/issues/58, but it shows the language in long form. I don't remember why I did it this way, must have seen it _somewhere_.

**Reply by WhoMI7** [reply]
> I see that we are forcing it to be uppercase with `track.lang:upper()`. I think @christoph-heinrich made it this way, but regardless, it's the most important piece of information in that hint that everyone is looking for, so I don't see an issue with making it more prominent by uppercasing it.

Prominent, maybe. Odd one out for sure. I don't see why anyone would look at the `lang` instead of just looking at the name of sub/audio and play "guess the language game in their mind (Like what could `enm` be instead of looking at the name: Honorifics)." Though its fine I guess since I can change it with `track.lang:lower()`

**Reply by hooke007** [reply]
> must have seen it somewhere.

yes, you could see it at many places.

https://support.isan.org/hc/en-us/articles/360012734159-List-of-ISO-639-2-Language-Codes
